speedviewlib/src/main/java/com/github/anastr/speedviewlib/RaySpeedometer.kt

```diff
@@ -104,12 +105,21 @@ open class RaySpeedometer @JvmOverloads constructor(
         val a = context.theme.obtainStyledAttributes(attrs, R.styleable.RaySpeedometer, 0, 0)
 
         rayPaint.color = a.getColor(R.styleable.RaySpeedometer_sv_rayColor, rayPaint.color)
-        val degreeBetweenMark = a.getInt(R.styleable.RaySpeedometer_sv_degreeBetweenMark, this.degreeBetweenMark)
-        val rayMarkWidth = a.getDimension(R.styleable.RaySpeedometer_sv_rayMarkWidth, rayMarkPaint.strokeWidth)
+        val degreeBetweenMark =
+            a.getInt(R.styleable.RaySpeedometer_sv_degreeBetweenMark, this.degreeBetweenMark)
+        val rayMarkWidth =
+            a.getDimension(R.styleable.RaySpeedometer_sv_rayMarkWidth, rayMarkPaint.strokeWidth)
         rayMarkPaint.strokeWidth = rayMarkWidth
         activeMarkPaint.strokeWidth = rayMarkWidth
-        speedBackgroundPaint.color = a.getColor(R.styleable.RaySpeedometer_sv_speedBackgroundColor, speedBackgroundPaint.color)
+        speedBackgroundPaint.color = a.getColor(
+            R.styleable.RaySpeedometer_sv_speedBackgroundColor,
+            speedBackgroundPaint.color
+        )
         withEffects = a.getBoolean(R.styleable.RaySpeedometer_sv_withEffects, withEffects)
+        rayStartSpeed =
+            a.getFloat(R.styleable.RaySpeedometer_sv_rayStartSpeed, Float.NEGATIVE_INFINITY).let {
+                if (it == Float.NEGATIVE_INFINITY) null else it
```

Comment:
This is kind of hacky but I couldn't find a good way in the API to allow null values.